DEVX-4960 Update Enable Audio Guide (#3469)
Nexmo/nexmo-developer: _partials/stitch/guides/enable-audio/java.md

### Request permission on application start

Request permissions inside `onViewCreated` method inside `LoginFragment` class:

``` java
@Override
public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
    // ...

    String[] callsPermissions = { Manifest.permission.RECORD_AUDIO };
    ActivityCompat.requestPermissions(this, callsPermissions, 123);
}
```

## Add audio UI

> You will now need to add two buttons for the user to enable and disable audio. Open `app/src/main/res/layout/fragment_chat.xml` file and add two new buttons (`enableMediaButton` and `disableMediaButton`) inside `chatContainer`:

Reviewer: ```suggestion
Add two buttons for the user to enable and disable audio. Open `app/src/main/res/layout/fragment_chat.xml` file and add two new buttons (`enableMediaButton` and `disableMediaButton`) inside `chatContainer`:
```

        app:layout_constraintTop_toBottomOf="parent"
        tools:visibility="visible">

    <Button
            android:id="@+id/enableMediaButton"
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            app:layout_constraintBottom_toTopOf="@id/conversationEventsScrollView"
            app:layout_constraintLeft_toLeftOf="parent"
            app:layout_constraintTop_toTopOf="parent"
            android:text="Enable Audio" />

    <Button
            android:id="@+id/disableMediaButton"
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            app:layout_constraintBottom_toTopOf="@id/conversationEventsScrollView"
            app:layout_constraintLeft_toLeftOf="parent"
            app:layout_constraintTop_toTopOf="parent"
            android:visibility="gone"
            android:text="Disable Audio"
            tools:visibility="visible"/>
<!--...-->
```

Now you need to make sure that these buttons are accessible in the fragment. Add two new properties in the `ChatFragment` class:

```java
private Button enableMediaButton;
private Button disableMediaButton;
```

> Retrieve the buttons references by adding `findViewById` calls in the `onViewCreated` method:

Reviewer: Needs to be either "retrieve the button references" or "retrieve the buttons' references"

}
```

> Add listeners to the buttons inside `onViewCreated` method:

Reviewer: ```suggestion
Add click event listeners for the buttons, inside the `onViewCreated` method:
```

}
```

> The `conversationEvents` observer inside `Chatragment` have to support newly added `NexmoMediaEvent` type. Add new branch to the if statement:

Reviewer: ```suggestion
The `conversationEvents` observer inside `ChatFragment` must support the `NexmoMediaEvent` type. Add a new branch to the if statement:
```
